include/swift/AST/ASTContext.h

```diff
@@ -865,6 +865,12 @@ class ASTContext {
   
   /// Whether our effective Swift version is in the Swift 3 family
   bool isSwiftVersion3() const { return LangOpts.isSwiftVersion3(); }
+  
+  /// Whether our effective Swift version is in the Swift 4 family
+  bool isSwiftVersion4() const { return LangOpts.isSwiftVersion4(); }
+  
+  /// Whether our effective Swift version is in the Swift 5 family
+  bool isSwiftVersion5() const { return LangOpts.isSwiftVersion5(); }
```

Comment:
Fair point about the <, <=, >, >=.  Removing isSwiftVersion4, isSwiftVersion5 from pull request now.